packages/next/cli/next-start.ts

```diff
@@ -58,10 +60,22 @@ const nextStart: cliCommand = (argv) => {
     port = 0
   }
 
+  const keepAliveTimeoutArg: number | undefined = args['--keepAliveTimeout']
+  let keepAliveTimeout: number | undefined
+
+  if (keepAliveTimeoutArg) {
```

Comment:
I removed the input-type validation (`args` module seems to already take care of this) and replaced it with some input-range validation. Then input-range validation clamps the `--keepAliveTimeout` value to be between 0 and Infinity-1, inclusive. 

There doesn't seem to be a way to write a test for this since 1) the invalid-range validation will not throw errors and 2) the underlying `http` server is not exposed from `startServer`. If the lack of tests for this validation will hold up the PR, I can revert the commit that introduced the range-validation. Let me know.

It would be nice to get this PR in in the next canary release. Let me know if you guys need something more, @styfle  @timneutkens

Reply:
Can you add a test to confirm it works properly and won't regress in the future?